vividus/src/main/java/org/vividus/runner/BddStepPrinter.java

```diff
@@ -68,15 +71,23 @@ public static void main(String[] args) throws ParseException, IOException
         Vividus.init();
         Set<Step> steps = getSteps();
 
+        int maxLocationLength = steps.stream().map(Step::getLocation).mapToInt(String::length).max().orElse(0);
+
+        List<String> stepsLines = steps.stream().map(s -> String
+                .format("%-" + (maxLocationLength + 1) + "s%-24s%s %s", s.getLocation(),
```

Comment:
why not %d and then pass (maxLocationLength + 1) as param?

Reply:
do you propose to use `String.format` twice?